tests/pytest_tests/system_tests/test_core/test_footer.py

```diff
@@ -117,3 +118,21 @@ def test_footer_history(wandb_init, check_output_fn):
     run.log(dict(a=3))
     run.finish()
     check_output_fn(exp_summary=[], exp_history=["a", "d", "🚀"])
+
+
+def test_footer_job_output(wandb_init, capsys, monkeypatch):
+    """Test that footer includes job info when a job is created."""
+    monkeypatch.setenv("WANDB_DOCKER", "hello-world")  # Needed to trigger job creation.
+    run = wandb_init()
+    run.log({"a": 1})
+    run.finish()
+    captured = capsys.readouterr()
+    lines = captured.err.splitlines()
+    for line in lines:
+        if re.match(
+            r"wandb: ⚡️ View job at http://localhost:8080/user-\w+-\w+/uncategorized/jobs/[\w=]+/version_details/latest",
```

Comment:
I'm seeing some really weird behavior actually. Right now we are limited to what is returned from ArtifactSaver.save https://github.com/wandb/wandb/blob/700ed2305c8cc57e46d1aa2d94a67072e50dd013/wandb/sdk/internal/sender.py#L1495

 Seems like sometimes this returns "latest" and sometimes it returns a version index. From what I can tell: if the artifact version already exists it returns the index alias (e.g v2), otherwise it returns latest.

Reply:
Mmm, that's not great, is the length ever > 1? Can we validate this in the core repo?

Reply:
https://github.com/wandb/wandb/blob/3dc3fe5c4b01d18f248cc6f304bb0cf78c7e399b/wandb/sdk/internal/internal_api.py#L3168-L3178 the client does some stuff here to set its version key (which we use to print the link) to the version number if the version number is included in this other part of the response. But for some reason the version number isn't included in the response for v0 of the artifact, not sure why that is. I'm taking a look at the mutation but I don't have my bearings that well so it will take a bit

Reply:
Then let's swap latest for v0, if multiple people run code that makes the same job collection, then latest may misdirect some users